src/hooks/useTokenIdentityRegistryAgents.ts

```diff
@@ -34,6 +37,33 @@ export function useTokenIdentityRegistryAgents(registry?: EvmAddress) {
     }
   }, [registry]);
 
+  const filteredNotAgentsYetQueryResult = useQueries({
+    queries: (identityItems ?? []).map((agent) => ({
```

Comment:
we already pass mapped out `identityItems` from above, why add converting to empty array again?